pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -30,7 +30,9 @@ import (
 	"k8s.io/kubernetes/pkg/credentialprovider"
 )
 
+const chinaRegionPrefix = "cn-"
 const registryURLTemplate = "*.dkr.ecr.%s.amazonaws.com"
+const chinaRegistryURLTemplate = "*.dkr.ecr.%s.amazonaws.com.cn"
```

Comment:
Thanks, it just looked weird, given that the site is on amazonaws.cn...

I kinda verified that at least the domain seems to resolve and point to a proxy to somewhere...
```